src/controller/sessionController.js

```diff
@@ -289,7 +291,8 @@ export async function getSessionState(req, res) {
 
 export async function getQrCode(req, res) {
   try {
-    const img = Buffer.from(req.client.qrcode.replace(/^data:image\/(png|jpeg|jpg);base64,/, ''), 'base64');
+    const qr = await QRCode.toDataURL(req.client.urlcode);
```

Comment:
I don't know if this is effected too with the same error as above, as I didn't test adding a new session, but I guess it's find, as here you're requesting the QRcode, and if it's null it should crash anyway.